ykpack/Cargo.toml

```
@@ -3,6 +3,7 @@ name = "ykpack"
 version = "0.1.0"
 authors = ["Edd Barrett <vext01@gmail.com>"]
 edition = "2018"
+license = "Apache-2.0/MIT"
```

https://doc.rust-lang.org/stable/cargo/reference/manifest.html#the-license-and-license-file-fields

```suggestion
license = "MIT OR Apache-2.0"
```

---

Agreed, though I'd prefer it to be alphabetical (in line with our other crates):

```
license = "Apache-2.0 OR MIT"
```

---

I actually copied this change from grmtools, so that must be wrong too?
https://github.com/softdevteam/grmtools/blob/721f46714f49d417111aff152ae5133117b247af/cfgrammar/Cargo.toml#L9

@ltratt I looked at yksom first, but this too is missing a license line.